atlp-rwanda/e-commerce-bitcrafters-bn
# #[187364724] - A successfully registered user should receive an email to verify their account email

**What does this PR do**
Users need to verify their email to avoid spams

**Description of Task to be completed?**
- Added nodemailer to help in sending emails
- Created a reusable component for sending emails

 **How should this be manually tested?**
- You can use postman to test it by going to my-app-backend-url/register
- Add your real email address and all necessary information
- If successful, check your gmail or any email platform you use 
- Click on the verification link provided

 **What are the relevant pivotal tracker/Trello stories?**
#187364724
